docs/testplan/Add-cluster-with-GCU.md

```diff
@@ -0,0 +1,127 @@
+# Add cluster with Generic Config Updater (GCU) Test Plan
+
+## Overview
+
```

Comment:
Here in the overview section can you give some details on Add cluster -- where we add a new T1 device set under T2 and mention the list of tables which we are updating like the PORT, PORTCHANNEL, PORTCHANNEL_MEMBER etc.